.github/workflows/linux-nightly.yml

```diff
@@ -0,0 +1,62 @@
+name: TorchArrow Nightly Build and Test
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  container:
+    runs-on: ubuntu-latest
+    container: prestocpp/velox-velox-torcharrow:kpai-20220301
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - 3.7
+          - 3.8
+          - 3.9
+    steps:
+      - name: Print CPU info
+        run: cat /proc/cpuinfo
+
+      - name: Setup Python environment
+        uses: actions/setup-python@v2
```

Comment:
Note all the CPython extension are built against Python 3.10, despite you set up Conda with Python 3.7/3.8/3.9:

```
[164/165] Linking CXX shared module csrc/velox/_torcharrow.cpython-310-x86_64-linux-gnu.so
```

Python 3.10 is indeed contained in the PATH and seems to be picked up with higher priority:
```
PATH='\''/opt/conda/bin:/opt/conda/condabin:/opt/conda/envs/ta_python_/bin:/opt/conda/condabin:/__t/Python/3.10.2/x64/bin:/__t/Python/3.10.2/x64:/opt/rh/devtoolset-10/root/usr/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'\''
```

The Python 3.10 seems to be setup by the Python environment here: 
```
/usr/bin/docker exec  cfcbe2eacbad[6](https://github.com/facebookresearch/torcharrow/runs/5485875682?check_suite_focus=true#step:4:6)31b8a29de181d67682baa7806e69c232673ab8a5a5328fcf2d9 sh -c "cat /etc/*release | grep ^ID"
Successfully setup CPython (3.10.2)
```

Reply:
> The Python 3.10 seems to be setup by the Python environment here. Do we need it (since IIUC, we actually want the Python environment brought by Conda right? )

Agree. This step should be removed.